src/components/pages/InstructorHome/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import InstructorSidebar from './InstructorSidebar';
 import InstructorCalender from './InstructorCalender';
 import InstructorProfile from './InstructorProfile';
-
+import { dummyData } from '../../../dummyData';
```

Comment:
It's nice to go ahead and get this done. I wonder if, as we progress through the project, if we will convert to redux at some point?

Reply:
Yes, once we get data from backend database, we will need to use redux to manage states.